### DISCUSSION TITLE
access speaker diarization model offline

### DISCUSSION THREAD

**Discussion by huks0** [pyannote/pyannote-audio#1103] [Development]
I want to use pyannote speaker diarization offline.

Is there a way to retrieve the and use model without connecting and downloading from huggingface?

**Comment by muhammad-usama-aleem** [comment]
@huks0 Did you find any way to do it? I am also looking for answer of the same issue

**Comment by mezaros** [comment]
If I could upvote this a thousand times, I would. What a great project that opened up tons of possibilities for me in the past few weeks. Why the sudden gatekeeping? Hope we can figure this part out, and host it all locally.

**Comment by hbredin** [comment]
I have just updated the [FAQ](https://github.com/pyannote/pyannote-audio/blob/develop/FAQ.md#can-i-use-gated-models-and-pipelines-offline) with instructions on how to use pretrained models and pipelines offline.

**Reply by k3ybladewielder** [reply]
> I have just updated the [FAQ](https://github.com/pyannote/pyannote-audio/blob/develop/FAQ.md#can-i-use-gated-models-and-pipelines-offline) with instructions on how to use pretrained models and pipelines offline.

@hbredin The diarization model is tha same of segmentation or we need to pickle the diarization model (theres no model on file session) and then use the config.yaml of the diarization model ?

**Reply by egshiu** [reply]
After editing the config.yaml, where do we put that file in order for it to load the pipeline correctly?
Also, the links you provided no longer have the pytorch_model.bin file. Is this a permanent removal?
And finally, just wanted to double-check, for the [Applying a Pipeline tutorial](https://github.com/pyannote/pyannote-audio/blob/develop/tutorials/applying_a_pipeline.ipynb), even though we're trying to use the offline model for the diarization, we still download the voice activity detection config.yaml for that?
